content/bdk-cli/installation.md

```diff
@@ -34,59 +34,43 @@ Once Cargo is installed, you can proceed to install the interactive `bdk-cli` to
 the GitHub repository, by running:
 
 ```bash
-cargo install --git https://github.com/bitcoindevkit/bdk-cli --features=esplora
+# all features
+cargo install --git https://github.com/bitcoindevkit/bdk-cli --features="compact_filters,esplora"
```

Comment:
I think you should take our `compact_filters` for now since it requires more setting to make it work (the below example command would fail), and add the `compiler` optional feature 
```suggestion
cargo install --git https://github.com/bitcoindevkit/bdk-cli --features="esplora,compiler"
```